Assets/_Scripts/Party.cs

```diff
@@ -5,35 +5,73 @@
 
 public class Party : MonoBehaviour {
     public PlayerQueue playerQueue;
-    public GameObject playerPrefab;
+    public Player heroPrefab;    // Optional TODO: Dependent on streamer
```

Comment:
Maybe we should move the player prefabs into PlayerCreationData?

Reply:
But then again it might be hard to reference without the inspector...

Reply:
Prefabs will be dependent on the Job later. We should include the prefab in the JobLists later.

Reply:
This isn't all that hard to do. Maybe I'll do it now